listings/ch00-introduction/constructor/src/lib.cairo

```diff
@@ -0,0 +1,12 @@
+mod constructor {
+    use starknet::ContractAddress;
+
+    struct Storage {
+        names: LegacyMap::<ContractAddress, felt252>, 
+    }
+
+    #[constructor]
+    fn constructor(_name: felt252, _address: ContractAddress) {
```

Comment:
An emerging Cairo convention is to have the storage variables prefixed with a `_` and function parameters without any prefix